KubeArmor/core/karmorprobedata.go

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package core
+
+import (
+	"encoding/json"
+	"io/ioutil"
```

Comment:
Please refer https://go.dev/doc/go1.16#ioutil for switching ioutil to suggested APIs.